README.md

```diff
@@ -55,11 +55,12 @@
   * Option to specify expected `Content-Type` when response `Content-Type` header missing. Refer to [#92](https://github.com/go-resty/resty/issues/92)
   * Resty design
     * Have client level settings & options and also override at Request level if you want to
-    * Request and Response middlewares
+    * Request and Response middleware
     * Create Multiple clients if you want to `resty.New()`
     * Supports `http.RoundTripper` implementation, see [SetTransport](https://godoc.org/github.com/go-resty/resty#Client.SetTransport)
     * goroutine concurrent safe
     * Resty Client trace, see [Client.EnableTrace](https://godoc.org/github.com/go-resty/resty#Client.EnableTrace) and [Request.EnableTrace](https://godoc.org/github.com/go-resty/resty#Request.EnableTrace)
+      * Since v2.4.0, `RequestAttempt` value supported in trace info also Request instance contains `Attempt` attribute
```

Comment:
Rewording suggestion:

Since v2.4.0, trace info contains a `RequestAttempt` value, and the Request object contains an `Attempt` attribute

Reply:
Thanks, @moorereason, I will update the readme with suggestion on the next round. I have to do a few other things.